src/client/Microsoft.Identity.Client/Cache/CacheTypeUsed.cs

```diff
@@ -10,21 +10,25 @@
 namespace Microsoft.Identity.Client.Cache
 {
     /// <summary>
-    /// Identifies the type of cache used when accessing the cache.
+    /// Identifies the type of cache used when accessing the cache. Cache implementations must provide this.
     /// </summary>
-    public enum CacheTypeUsed
+    public enum CacheLevel
     {
         /// <summary>
-        /// Specifies that no cache type is used
+        /// Specifies that the cache level used is unknown.
         /// </summary>
-        None = 0,
+        Unknown = 0,
         /// <summary>
         /// Specifies if the L1 cache is used
         /// </summary>
         L1Cache = 1,
         /// <summary>
         /// Specifies if the L2 cache is used
         /// </summary>
-        L2Cache = 2
+        L2Cache = 2,
+        /// <summary>
+        /// Specifies if both the L1 and L2 caches are used
+        /// </summary>
+        L1AndL2Cache = 3
```

Comment:
As discussed offline, this is not possible. We are only interested in READs, and a READ operation can happen in L1 or in L2, but not in both.